widget/embedded/src/constants/languages.ts

```diff
@@ -41,6 +66,43 @@ export const LANGUAGES: LanguageItem[] = [
   { title: 'Greek', label: 'Grieks', local: 'el', SVGFlag: Greece },
   { title: 'Italian', label: 'Italiana', local: 'it', SVGFlag: Italian },
   { title: 'Polish', label: 'Polski', local: 'pl', SVGFlag: Poland },
+
+  { title: 'Afrikaans', label: 'Afrikaans', local: 'af', SVGFlag: SouthAfrica },
+  { title: 'Arabic', label: 'عربي', local: 'ar', SVGFlag: SaudiArabia },
+  { title: 'Bengali', label: 'বাংলা', local: 'bn', SVGFlag: Bengali },
+  { title: 'Catalan', label: 'català', local: 'ca', SVGFlag: Catalonia },
+  { title: 'Danish', label: 'dansk', local: 'da', SVGFlag: Denmark },
+  { title: 'Hindi', label: 'हिंदी', local: 'hi', SVGFlag: India },
+  { title: 'Hungarian', label: 'magyar', local: 'hu', SVGFlag: Hungary },
+  { title: 'Indonesian', label: 'Indonesia', local: 'id', SVGFlag: Indonesian },
+  { title: 'Korean', label: '한국인', local: 'ko', SVGFlag: Korea },
+  { title: 'Lithuanian', label: 'lietuvių', local: 'lt', SVGFlag: Lithuania },
+  { title: 'Malay', label: 'Melayu', local: 'ms', SVGFlag: Malay },
+  { title: 'Filipino', label: 'Filipino', local: 'fil', SVGFlag: Philippines },
+  { title: 'Serbian', label: 'српски', local: 'sr', SVGFlag: Serbia },
+  { title: 'Slovak', label: 'slovenský', local: 'sk', SVGFlag: Slovakia },
+  { title: 'Swahili', label: 'kiswahili', local: 'sw', SVGFlag: Swahili },
+  { title: 'Thai', label: 'แบบไทย', local: 'th', SVGFlag: Thai },
+  { title: 'Turkish', label: 'Türkçe', local: 'tr', SVGFlag: Turkish },
+  { title: 'Urdu', label: 'اردو', local: 'ur', SVGFlag: Pakistan },
+  {
+    title: 'Vietnamese',
+    label: 'Tiếng Việt',
+    local: 'vi',
+    SVGFlag: Vietnamese,
+  },
+  {
+    title: 'Greek(cyprus)',
+    label: 'Ελληνικά(Κύπρος)',
```

Comment:
```suggestion
    title: 'Greek',
    label: 'Ελληνικά',
```

Reply:
We have two separate Greek languages, so they are named this way to distinguish them